src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
+  }
+  :last-child {
+    margin-right: 0;
```

Comment:
It seems you have a small mistake in your selector. The `:last-child` pseudo-class should be used with the `&__star` selector to target the last star, not with the `.stars` block itself. It should be `&__star:last-child` to ensure that the last star in the `.stars` element has no right margin.